web-components/infinite-scroll/src/infinite-scroll.ts

```diff
@@ -0,0 +1,270 @@
+      getSortedRowModel: getSortedRowModel(),
+      getCoreRowModel: getCoreRowModel(),
+    });
+    const { rows } = table.getRowModel();
+    const virtualizer = controller.getVirtualizer();
+
+    if (this.data.length === 0 && this.loading) {
+      return html`loading...`;
+    }
+
+    return html`
+      <div
+        ${ref(this.tableContainerRef)}
+        class="container"
+        @scroll=${this.fetchMoreOnBottomReached}
+        style="${styleMap({
+          overflow: 'auto', //our scrollable table container
+          position: 'relative', //needed for sticky header
+          height: '540px', //should be a fixed height
+        })}">
+        <cds-table style="display: grid">
+          <cds-table-head
+            style="${styleMap({
+              display: 'grid',
+              position: 'sticky',
+              top: 0,
+              zIndex: 1,
+            })}">
+            ${repeat(
+              table.getHeaderGroups(),
+              (headerGroup) => headerGroup.id,
+              (headerGroup) =>
+                html`<cds-table-header-row
+                  style="${styleMap({ display: 'flex', width: '100%' })}">
+                  ${repeat(
+                    headerGroup.headers,
+                    (header) => header.id,
+                    (header) =>
+                      html` <cds-table-header-cell
+                        style="${styleMap({
+                          display: 'flex',
+                          width: `${header.getSize()}px`,
+                          alignItems: 'center',
+                        })}">
+                        ${header.isPlaceholder
+                          ? null
+                          : flexRender(
+                              header.column.columnDef.header,
+                              header.getContext()
+                            )}
+                      </cds-table-header-cell>`
+                  )}</cds-table-header-row
+                >`
+            )}
+          </cds-table-head>
+          <cds-table-body
+            style=${styleMap({
+              display: 'grid',
+              height: `${virtualizer.getTotalSize()}px`, //tells scrollbar how big the table is
+              position: 'relative', //needed for absolute positioning of rows
+            })}>
+            ${repeat(
+              virtualizer.getVirtualItems(),
+              (item) => item.key,
+              (item) => {
+                const row = rows[item.index] as Row<Resource>;
+                const displaySkeleton =
+                  this.loading && this.data.length - 1 === item.index;
+                return html`
+                  <cds-table-row
+                    style=${styleMap({
+                      display: 'flex',
+                      position: 'absolute',
+                      transform: `translateY(${item.start}px)`,
+                      width: '100%',
+                    })}
+                    ${ref((node) => virtualizer.measureElement(node))}>
```

Comment:
This line is causing a lot of warnings being logged to the console on every scroll event. Can you check that out?
![image](https://github.com/user-attachments/assets/57be4df0-51c0-46da-8375-971b034c6dac)

Reply:
you bet 👍